src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2716,6 +2716,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Select
         End Function
 
+        Public Overrides Function GetRequiredLanguageVersion(diagnostic As Diagnostic) As String
+            ' Upgrading project based on language versions reported in diagnostics isn't supported in VB at this point
```

Comment:
This statement seems at odds with the definition of the API: "this diagnostic indicates we need a higher language version" and "the IDE supports a code fix" are two separate things.